### PR TITLE
Add missing copyright headers

### PR DESCRIPTION
**Pull request by sebwoj**
Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
For each of these, we should use the git logs to find out the date the the file started, and also use the author list to find out if any non Red Hat employees contributed (because I usually wrote "Red Hat, Inc. and others" if the file has contributions from others. Otherwise our copyright claims aren't quite correct.

**Comment by sebwoj** [comment]
Files `bodhi/tests/client/__init__.py`(RH + others) and `bodhi/tests/server/scripts/test_clean_old_mashes.py`(only RH) where created in 2016 from scratch. But other files where renamed in 2016, and `git log -- file_name` indicate that first commit for each file was in 2016 and `git blame` shows changes from previous years(changes in files before rename).

File: bodhi/tests/server/functional/test_packages.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-
+# Copyright © 2016-2019 Red Hat, Inc.
```
Comment (bowlofeggs) [inline bodhi/tests/server/functional/test_packages.py]: No commits yet this year.

```suggestion
# Copyright © 2015-2018 Red Hat, Inc.
```

File: bodhi/tests/server/functional/test_users.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Copyright © 2016-2019 Red Hat, Inc.
```
Comment (bowlofeggs) [inline bodhi/tests/server/functional/test_users.py]: No commits yet this year.

```suggestion
# Copyright © 2014-2018 Red Hat, Inc. and others
```

File: bodhi/tests/server/scripts/test_clean_old_mashes.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# Copyright © 2016-2019 Red Hat, Inc.
```
Comment (bowlofeggs) [inline bodhi/tests/server/scripts/test_clean_old_mashes.py]: No commits yet this year.

```suggestion
# Copyright © 2016-2018 Red Hat, Inc.
```

File: bodhi/tests/server/test_schemas.py [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# Copyright © 2016-2019 Red Hat, Inc.
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_schemas.py]: No commits yet this year.

```suggestion
# Copyright © 2014-2018 Red Hat, Inc. and others.
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
You can ask git log to show you the log for files that were moved prior to their move by using it's ```--follow``` flag. I'm not sure that Mathiue Bridon was/is a Red Hatter, so I changed a few of these to say "and others" as well.

**Comment by sebwoj** [comment]
I'm not sure, but since I'm(as a non RH) changing these files(only copyright), should I add "and others" in the copyright hearer? Also you mentioned that each file should have "-2018" year(if no commits in 2019), so is adding "copyright header" to a file some kind of exception? I'm asking because I have no experience in this topic.

**Comment by bowlofeggs** [comment]
On Fri, 2019-03-08 at 06:47 -0800, Sebastian Wojciechowski wrote:
> I'm not sure, but since I'm(as a non RH) changing these files(only
> copyright), should I add "and others" in the copyright hearer? Also
> you mentioned that each file should have "-2018" year(if no commits
> in 2019), so is adding "copyright header" to a file some kind of
> exception? I'm asking because I have no experience in this topic.

I don't think changing the copyright claim itself is copyrightable, but
I am also not a lawyer.

In general though, I tend to use the formal "Red Hat, Inc. and others"
when people who are not Red Hat employees contribute patches. Since
they agree to the DCO as documented in the contributor guidelines, they
are licensing the patch to the project under the GPLv2+ (since that's
how the project is licensed).

In general, please do feel free to add the "and others" to files you
modify if they don't already have it. I just don't think the copyright
block itself is copyrighted, though again, I'm not a lawyer ☺

**Review by bowlofeggs — APPROVED** [review]
Thanks!

**Pull request merged**